### PR TITLE
Avoid duplicate query blocks

### PR DESCRIPTION
**Pull request by zjg555543**
**Issue:** 
Occasionally receiving duplicate log events.

**Fixed:**
- `lastCheckedL1Block` means it has already been checked in the previous cycle.
- It should not be checked again in the new cycle.

### CODE REVIEW

File: zk/syncer/l1_syncer.go [hunk 1]
```diff
@@ -302,7 +302,7 @@ func (s *L1Syncer) getLatestL1Block() (uint64, error) {
 }
 
 func (s *L1Syncer) queryBlocks() error {
-	startBlock := s.lastCheckedL1Block.Load()
+	startBlock := s.lastCheckedL1Block.Load() + 1
 
 	log.Debug("GetHighestSequence", "startBlock", s.lastCheckedL1Block.Load())
```
Comment (V-Staykov) [inline zk/syncer/l1_syncer.go]: Let's also make this use start block and not the atomic

Comment (zjg555543) [reply zk/syncer/l1_syncer.go]: Done

File: zk/syncer/l1_syncer.go [hunk 2]
```diff
@@ -302,9 +302,9 @@ func (s *L1Syncer) getLatestL1Block() (uint64, error) {
 }
 
 func (s *L1Syncer) queryBlocks() error {
-	startBlock := s.lastCheckedL1Block.Load()
+	startBlock := s.lastCheckedL1Block.Load() + 1
```
Comment (mandrigin) [inline zk/syncer/l1_syncer.go:308]: why +1 here?

**Review by mandrigin — CHANGES_REQUESTED** [review]
questions

File: zk/syncer/l1_syncer.go (hunk 2 above)
Comment (zjg555543) [reply zk/syncer/l1_syncer.go:308]: It occasionally receives duplicate log events. `lastCheckedL1Block` indicates that it has already been checked in the previous cycle. Therefore, it should not be checked again in the new cycly and +1 here.

**Comment by zjg555543** [comment]
> questions

@mandrigin @V-Staykov 
I think querying the duplicate block is the rootcase of [PR787](https://github.com/0xPolygonHermez/cdk-erigon/pull/787).
I encountered a similar issue during testing. The l1 info tree's update index will add+1 when recieving dumplicate, which leading the aggragor can't to verify.

**Review by V-Staykov — DISMISSED** [review]

File: zk/syncer/l1_syncer.go (hunk 2 above)
Comment (mandrigin) [reply zk/syncer/l1_syncer.go:308]: can you add a comment for that?

Comment (zjg555543) [reply zk/syncer/l1_syncer.go:308]: Done

**Review by V-Staykov — DISMISSED** [review]

**Review by mandrigin — DISMISSED** [review]

**Comment by mandrigin** [comment]
@zjg555543 you need to sign you commits now -- new policy from Polygon

**Review by mandrigin — APPROVED** [review]

**Pull request merged**